assets/scripts/evolution/svg_pagination.js

```diff
@@ -0,0 +1,16 @@
+export default function() {
+	(function() {
+		var elems = document.querySelectorAll('ul.evo_c-svg-dot--default li'),
+		makeActive;
+
+		makeActive = function() {
```

Comment:
hello @jmatembu , I met a problem, when I push, it says my local  c-svg-pagination is behind the remote c-svg-pagination. So should I pull the remote branch firstly? Thanks!